tests/integration/backend/filestate/backend_azure_test.go

```diff
@@ -5,29 +5,32 @@ import (
 	"os/exec"
 	"testing"
 
+	"github.com/google/uuid"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func loginAndCreateStack(t *testing.T, cloudURL string) {
 	t.Helper()
+
+	stackName := uuid.New().String()
```

Comment:
Yeh lets try and reuse the random stack name functions that already exist

Reply:
TIL.  It's in a different package (seems like we only use it for automation api currently), and I don't think we have a good way for sharing these, so I just copy pasted it for now, since it's really simple.  If we end up needing it in more different places we can factor it out.